src/platform_projects/openthread-efr32-rcp.slcp

```diff
@@ -1,7 +1,7 @@
 project_name: openthread-efr32-rcp
```

Comment:
Should we rename this file/project to `openthread-efr32-rcp-uart.slcp` to be consistent with the new `openthread-efr32-rcp-spi.slcp` file/project.